numpy/core/numeric.py

```diff
@@ -3007,6 +3009,8 @@ class errstate(object):
     that context to execute with a known error handling behavior. Upon entering
     the context the error handling is set with `seterr` and `seterrcall`, and
     upon exiting it is reset to what it was before.
+    
+    In Python 3, errstate is also usable as a function decorator.
```

Comment:
Needs `..versionchanged: 1.14` before, or whatever the actual syntax is. Can we mention `ContextDecorator` here?